### PR TITLE
docs: Add necessary preceding requisite steps to installing FlutterFire CLI. i.e. Nodejs & Firebase CLI installation.

### PR DESCRIPTION
**Pull request by kagouda**
To solve this problem: 
https://github.com/FirebaseExtended/flutterfire/issues/7805
And I added the difference between FlutterFire CLI and Firebase CLI, because it's not the same thing.  Also the FlutterFire CLI depends on the Firebase CLI so it must be installed before you can use it

## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: docs/overview.mdx [hunk 1]
```diff
@@ -71,6 +71,13 @@ is backwards compatible with [manual-installation](./manual-installation.mdx).
 
 ### Using the CLI
 
+The FlutterFire CLI depends on the [Firebase CLI](https://firebase.google.com/docs/cli). If you haven't already done so, make sure you have the Firebase CLI installed on your machine. 
+If not, make sure you have [node.js](https://www.nodejs.org/) on your machine and Install the Firebase CLI via npm by running the following command :
```
Comment (russellwheatley) [inline docs/overview.mdx:75]:
```suggestion
You will also need [node.js](https://www.nodejs.org/) on your machine to Install the Firebase CLI via npm by running the following command: 
```

**Review by russellwheatley — CHANGES_REQUESTED** [review]
A few nits. Thanks.

**Comment by kagouda** [comment]
Thanks you @russellwheatley

**Comment by Salakar** [comment]
Could these changes be perhaps moved to /cli docs page instead?

**Comment by russellwheatley** [comment]
@kagouda, As @Salakar suggested, could you move these changes to the CLI page here: https://firebase.flutter.dev/docs/cli, please? Thanks 😄.

**Comment by kagouda** [comment]
@Salakar  it think it's ok now

File: docs/overview.mdx [hunk 2]
```diff
@@ -71,6 +71,8 @@ is backwards compatible with [manual-installation](./manual-installation.mdx).
 
 ### Using the CLI
```
Comment (russellwheatley) [inline docs/overview.mdx:72]:
```suggestion
### Using the FlutterFire CLI
```

File: docs/overview.mdx [hunk 3]
```diff
@@ -71,6 +71,8 @@ is backwards compatible with [manual-installation](./manual-installation.mdx).
...
 
+:::info The FlutterFire CLI depends on the [Firebase CLI](https://firebase.google.com/docs/cli).  If you haven't installed it yet, you can learn more about Firebase CLI in the [documentation](https://firebase.google.com/docs/cli). :::
```
Comment (russellwheatley) [inline docs/overview.mdx:74]:
```suggestion
:::info 

The FlutterFire CLI depends on the [Firebase CLI](https://firebase.google.com/docs/cli).  If you haven't installed it yet, you can learn more about Firebase CLI in the [documentation](https://firebase.google.com/docs/cli). 

:::
```

**Review by russellwheatley — CHANGES_REQUESTED** [review]
As long as @Salakar is happy with the link from `docs/overview.mdx`, and the tweaks I've requested, it LGTM.

**Comment by russellwheatley** [comment]
Hey @kagouda, we're happy with the changes, but could you make necessary updates so that the `website / validate` CI check passes? In a nutshell, change "behaviour" to "behavior" (US spelling) & add "npm" to the [dictionary](https://github.com/FirebaseExtended/flutterfire/blob/master/website/dictionary.js) file. Thanks.

**Comment by kagouda** [comment]
Hi @russellwheatley   . I have updated the dictionary and other . So I think it's OK. But where is the "behavior" that I should change "behavior" ?

**Comment by russellwheatley** [comment]
@kagouda, here is the CI test that is failing https://github.com/FirebaseExtended/flutterfire/runs/5112709183?check_suite_focus=true

You need to add "npm" to dictionary, not "node".  Thanks.

**Comment by kagouda** [comment]
@russellwheatley I've already updated the dictionary.

**Pull request merged**